File: fabric/src/main/java/com/selfdot/cobblemontrainers/fabric/CobblemonTrainersFabric.java
package com.selfdot.cobblemontrainers.fabric;

import com.selfdot.cobblemontrainers.CobblemonTrainers;
import net.fabricmc.api.ModInitializer;

public class CobblemonTrainersFabric implements ModInitializer {

    @Override
    public void onInitialize() {
        CobblemonTrainers.INSTANCE.initialize();
        System.out.println("CobblemonTrainers Fabric initialized");
    }

}

File: fabric/src/main/kotlin/com/selfdot/cobblemontrainers/fabric/CobblemonTrainersFabric.kt
package com.selfdot.cobblemontrainers.fabric

import com.selfdot.cobblemontrainers.CobblemonTrainers
import net.fabricmc.api.ModInitializer

class CobblemonTrainersFabric : ModInitializer {

    override fun onInitialize() {
        CobblemonTrainers.initialize()
        println("CobblemonTrainers Fabric initialized")
    }

}
